src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -313,12 +277,18 @@ private void UpdateCurrentConfiguration()
             }
             finally
             {
-                _syncAfter = DateTimeUtil.Add(DateTime.UtcNow, AutomaticRefreshInterval);
                 Interlocked.Exchange(ref _configurationRetrieverState, ConfigurationRetrieverIdle);
             }
 #pragma warning restore CA1031 // Do not catch general exception types
         }
 
+        private void ValidConfigurationObtained(T configuration)
```

Comment:
This name seems off. Maybe something like `StoreValidConfiguration`?
```suggestion
        private void StoreValidConfiguration(T configuration)
```